sdk/communication/communication-chat/test/internal/utils/mockClient.ts

```diff
@@ -162,6 +172,24 @@ export const mockMessageWithAttachment: RestModel.ChatMessage = {
   metadata: { tags: "tag" },
 };
 
+export const mockMessageWithDLP: RestModel.ChatMessage = {
+  id: "id",
+  type: "text",
+  version: "version",
+  sequenceId: "sequenceId",
+  content: {
+    message: "content",
+    topic: "topic",
+    attachments: [],
+  },
+  createdOn: new Date("2020-06-26T18:06:06Z"),
+  senderDisplayName: "senderDisplayName",
+  senderCommunicationIdentifier: mockCommunicationIdentifier,
+  deletedOn: new Date("2020-06-26T18:06:06Z"),
+  metadata: { tags: "tag" },
+  policyViolation: { state: "contentBlocked"}
```

Comment:
Should also have a mock for warning as well

Reply:
Also a test for without `policyViolation`